src/run_server.sh

```diff
@@ -97,12 +116,22 @@ function update_folder_permissions() {
     printf "\n### Folder Permissions updated.\n"
 }
 
+# start the server to generate configuration file - Prep for post install parameters [30 second start then kill]
+function first_start_config_ops()
+{
+	if[! -d "/root/Zomboid/"]
+	{
+		printf "\n### First Boot Detected - Burst Startup to create config files.\n"
+		start_server_initial
+		apply_postinstall_config
+	}
+}
 # Set variables for use in the script
 function set_variables() {
     printf "\n### Setting variables...\n"
 
     BASE_GAME_DIR="/home/steam/ZomboidDedicatedServer"
-    CONFIG_DIR="/home/steam/Zomboid/"
+    CONFIG_DIR="/root/Zomboid"
```

Comment:
We should be able to stick with the standard `steam` user for this.

Reply:
Folder is not created at location (/home/steam/Zomboid) need to figure out why

Reply:
Are you sure you are not doing any of the following?

- running Docker as root 
  - `sudo docker run` or `sudo docker-compose up`
- running the Docker as the root user by ID
  - `docker run ... user=0:0`

Those are the only two ways I can see files being created in the `/root` directory.

Reply:
Will validate this week